packages/cli/src/cmds/validator/keymanager/decryptKeystoreDefinitions/types.ts

```diff
@@ -0,0 +1,14 @@
+import {Logger} from "@lodestar/utils";
+import {LocalKeystoreDefinition} from "../interface.js";
+
+export type DecryptKeystoreWorkerAPI = {
+  decryptKeystoreDefinition({keystorePath, password}: LocalKeystoreDefinition): Promise<Uint8Array>;
+};
+
+export type KeystoreDecryptOptions = {
+  ignoreLockFile?: boolean;
```

Comment:
haven't tested this but renaming `force` --> `ignoreLockFile` likely breaks current `--force` CLI option

we just destructure args here, and top-level flag is not called `ignoreLockFile`

https://github.com/ChainSafe/lodestar/blob/a42fddd356c0511c686622d688a5133943b732a6/packages/cli/src/cmds/validator/signers/index.ts#L98-L99

1. Do we want to keep the name `force` for the CLI flag? will it ever do more than ignore lock files? and if not just rename?
2. Other option to 1., keep top-level name of CLI flag but use more descriptive and intuitive name like `ignoreLockFile`  downstream
3. I feel like we should explicitly assign args here to get type safety